src/Sarif/Errors.cs

```diff
@@ -509,6 +510,28 @@ public static RuntimeConditions LogUnhandledEngineException(IAnalysisContext con
             return RuntimeConditions.ExceptionInEngine;
         }
 
+        public static void LogIncompatibleRule(IAnalysisContext context, string ruleId, string incompatibleRuleId)
```

Comment:
>LogIncompatibleRule [](http://example.com/codeflow?start=27&length=19)

This should be plural in form, 'LogIncompatibleRules'.

Also, you haven't added the string value for this message as a comment in this code. Please observe other LogXXX members to see the pattern.

Reply:
fixed

Reply:
Nice!